goserver/server/server_test.go

```diff
@@ -76,3 +70,24 @@ func TestLaunchHandlerCheckISSAuthorization(t *testing.T) {
 		})
 	}
 }
+
+func TestHandleLaunch(t *testing.T) {
+	fhirURL := "https://authorized.fhir.com"
+	sm := session.NewInMemorySessionManager()
+	s := &Server{authorizedFHIRURL: fhirURL, sm: sm}
```

Comment:
Prefer using public APIs as much as possible in tests. It makes the tests more realistic and in this case less brittle.

Reply:
What do you mean by "public APIs"? Do you mean start a testing server and send HTTP request to it (instead of calling handler directly)?

Reply:
No sorry, I mean "public API" in the context of the Go package you are testing. So use NewServer() to create a server object instead of referencing private field directly.

Reply:
Ah got it.